app/controllers/katello/application_controller.rb

```diff
@@ -349,15 +169,13 @@ def render_bad_parameters(*args)
 
       if exception
         log_exception exception
-        notify.exception(message, exception)
       else
-        notify.error message
         Rails.logger.warn message
       end
 
       respond_to do |format|
         format.html do
-          render :template => 'katello/common/400', :layout => !request.xhr?, :status => status,
+          render :template => 'common/400', :layout => !request.xhr?, :status => status,
```

Comment:
@jlsherrill hmm I made the SyncManagementController sync method 
 raise a 400 by adding  ``` fail ::Katello::HttpErrors::BadRequest, "BOOOOOO!!!!"``` and hoping this would get called. Didnt seem to happen.. Instead a 500 got raised
```
2017-07-15T05:48:28 fcbf1ba9 [app] [D] Setting current location thread-local variable to none
2017-07-15T05:48:28 fcbf1ba9 [app] [I] Completed 500 Internal Server Error in 98ms (ActiveRecord: 5.2ms)
2017-07-15T05:48:28 fcbf1ba9 [app] [F] 
 | Katello::HttpErrors::BadRequest (BOOOOOO!!!!):
 |   /home/vagrant/katello/app/controllers/katello/sync_management_controller.rb:35:in `sync'
 |   app/controllers/concerns/application_shared.rb:15:in `set_timezone'
 |   app/controllers/concerns/foreman/controller/topbar_sweeper.rb:12:in `set_topbar_sweeper_controller'
 |   /home/vagrant/katello/lib/katello/params_parser_wrapper.rb:12:in `call'
 |   lib/middleware/catch_json_parse_errors.rb:8:in `call'
 |   lib/middleware/tagged_logging.rb:18:in `call'
 | 
```